tools/codegen/gentargets.py

```diff
@@ -16,12 +16,16 @@ def get_files_to_compile(json_data):
     for element in json_data:
         filename = element["file"]
         if not filename.endswith("_tests.cpp") and \
-            "third-party" not in filename and \
-            "generated" not in filename:
-            filename = filename.replace(REPO_ROOT_DIR + "/osquery/", "")
-            filename = filename.replace(REPO_ROOT_DIR + "/build/sync/code-analysis/", "")
+                "third-party" not in filename and \
+                "example" not in filename and \
+                "generated/gen" not in filename:
```

Comment:
this grabs the utils_amalgamated.cpp file, needed for basic utility tables (which are not broadcasted from core->extensions).